composer.json

```diff
@@ -29,20 +29,20 @@
         }
     },
     "require": {
-        "php": "^7.0 || ^5.6",
-        "laminas/laminas-stdlib": "^2.7 || ^3.0",
-        "laminas/laminas-zendframework-bridge": "^1.0"
+        "php": "^7.3",
+        "laminas/laminas-stdlib": "^3.2.1",
+        "laminas/laminas-zendframework-bridge": "^1.0.4"
     },
     "require-dev": {
-        "laminas/laminas-cache": "^2.6.1",
-        "laminas/laminas-coding-standard": "~1.0.0",
+        "laminas/laminas-cache": "^2.9.0",
+        "laminas/laminas-coding-standard": "~2.0.0",
         "laminas/laminas-config": "^2.6.0",
-        "laminas/laminas-db": "^2.9.2",
-        "laminas/laminas-filter": "^2.6.1",
+        "laminas/laminas-db": "^2.11.3",
+        "laminas/laminas-filter": "^2.9.4",
         "laminas/laminas-json": "^2.6.1",
-        "laminas/laminas-servicemanager": "^2.7.5 || ^3.0.3",
-        "laminas/laminas-view": "^2.6.3",
-        "phpunit/phpunit": "^6.2.1 || ^5.7.15"
+        "laminas/laminas-servicemanager": "^3.4.1",
+        "laminas/laminas-view": "^2.11.4",
+        "phpunit/phpunit": "^8.5.8"
```

Comment:
We can use only v9

Reply:
Will be fixed with one of the [next steps](https://github.com/laminas/laminas-paginator/projects/1).